src/main/java/org/myrobotlab/service/InMoov.java

```diff
@@ -1785,9 +1759,10 @@ public void loadCalibration(String calibrationFilename) {
   }
 
   public void saveCalibration() {
-    saveCalibration(CALIBRATION_FILE);
+    // saveCalibration(CALIBRATION_FILE);

```

Comment:
I use this.. let's not remove it.

Reply:
fine .. although mixing python generation with a Java service that is not related to Python at all gives me bumps ... I think it should use json in a declarative fashion to get back to a worky state